setup.py

```diff
@@ -6,11 +6,78 @@
 import os, sys
 from os import chdir, getcwd
 from os.path import abspath, dirname, split, samefile
-import shlex
+import shlex, glob
 from subprocess import check_output
 
 import re
 
+# Distribution target: 'all', 'ndt', 'nd'
+DIST_TARGET = 'all'
```

Comment:
It'd be nice if we could have this be a command line argument to the setup.py rather than something that has to be modified in the source files. That doesn't have to happen in this PR though.